app/assets/stylesheets/calagator/forms.scss

```diff
@@ -11,7 +11,9 @@
 
 ############################################################## */
 
-@import 'datepicker';
+/*
+ *= reuqire jquery-ui/datepicker
```

Comment:
Good catch on the typo! Too bad the tests can't catch it. :/

The limitation with the sprockets require syntax is that each sass file is evaluated in its own isolated context, so any sass variables defined don't propagate. However, that doesn't apply here, so I think this syntax is okay.

That said, it's simpler to not know or care about that limitation and simply use import everywhere. So I think that's a valid case for a future PR.